src/main.cpp

```diff
@@ -567,12 +568,14 @@ namespace {
 		}
 		return "";
 	}
+#endif
 
```

Comment:
This turns kinda messy with all those #ifdef's, a different solution would be nicer.

Reply:
How about moving `findProgram()`, `debuggerNames`, and `getDebuggerArgs()` inside the `use_debugger()`? It is using the `namespace` block currently, but now we can use lambda functions.

Reply:
I still don't think that's an improvement, sorry.
Do we really need to fix this warning?

Reply:
If unused function warning and unused variable warning is important, this is important. I just want to get rid of these warnings when compiling for Android.